memtier_benchmark.h

```diff
@@ -49,6 +49,8 @@ struct benchmark_config {
     unsigned int requests;
     unsigned int clients;
     unsigned int threads;
+    bool split;
```

Comment:
i'm not sure `split` is a good name.
btw, why do you need this parameter? i mean, why can check if a cpus list was provided and define the affinity accordingly?

Reply:
changed split to cpu_split.
in case that cpu-split was passed and no taskset, then the threads are run with affinity and the taskset is set to be all the available cpus in the system